app/src/main/java/org/oppia/app/topic/TopicFragment.kt

```diff
@@ -10,14 +10,21 @@ import javax.inject.Inject
 
 /** Fragment that contains tabs for Topic. */
 class TopicFragment : InjectableFragment() {
-  @Inject lateinit var topicFragmentPresenter: TopicFragmentPresenter
+  @Inject
+  lateinit var topicFragmentPresenter: TopicFragmentPresenter
+  private var topicId: String = ""
 
   override fun onAttach(context: Context?) {
     super.onAttach(context)
     fragmentComponent.inject(this)
   }
 
   override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-    return topicFragmentPresenter.handleCreateView(inflater, container)
+    @Suppress("NULLABILITY_MISMATCH_BASED_ON_JAVA_ANNOTATIONS")
+    topicId =
+      if (arguments != null && arguments!!.getString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY) != null) arguments!!.getString(
+        TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY
```

Comment:
This is creating confusion in brackets:

better way:

`if (arguments != null && arguments!!.getString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY) != null)`
`    arguments!!.getString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY) else ""`

Reply:
Resolved